tests/unit/model/designation-test.js

```diff
@@ -54,3 +54,191 @@ test('Relationships with other models', function(assert){
   assert.equal(relationshipOrdersPackage.key, 'ordersPackages');
   assert.equal(relationshipOrdersPackage.kind, 'hasMany');
 });
+
+test('check isLocalOrder', function(assert){
+  const model = this.subject({detailType: 'LocalOrder'});
+  assert.equal(model.get('isLocalOrder'), true);
+});
+
+test('check dispatchedItems returns items with sentOn not null', function(assert){
+  assert.expect(3);
+  const model = this.subject();
+  var store = this.store();
+  var item1 = null;
+  var item2 = null;
+  var item3 = null;
+  var despatchedItemsIds = null;
+
+  Ember.run(function(){
+    store.createRecord('item', { id: 1, sentOn: "12/07/2016" });
```

Comment:
@viveksriram1 , rather then hardcoding date can we use jQuery date object or **ember moment** here?
Also is it possible that we can directly assign this to item1?
Means item1 = store.createRecord('item', { id: 1, isDispatched: true });, if store.createRecord returns a model object